test/test_nn.py

```diff
@@ -15126,6 +15126,7 @@ def test_softmax_results(self, device, dtype):
     @onlyCUDA
     @dtypesIfCUDA(torch.float, torch.half)
     @largeTensorTest("20GB")
+    @largeTensorTest("40GB", "cpu")
```

Comment:
Is 40GB sufficient? The test was failing for me with 64GB host RAM.

Reply:
Some bad napkin math:
2200000000 elements in 32bit : ~8.4 GiB
==========================
xx on CPU : ~4.2GiB
xx.float() on CPU: ~8.4GiB
yy on CPU: ~8.4GiB
copying y to CPU for comparison: ~8.4GiB max?
xx.grad on CPU: ~8.4GiB
copying x.grad to CPU for comparison, ~8.4GiB max?
You're right, I think it could be more than 40GiB indeed...
though it seems like it would pass with 64GiB (free) host memory?